pkg/noderesourcetopology/score.go

```diff
@@ -135,7 +140,25 @@ func containerScopeScore(pod *v1.Pod, zones topologyv1alpha1.ZoneList, scorerFn
 	allocatablePerNUMA := createNUMANodeList(zones)
 
 	for i, container := range containers {
+		identifier := pod.Namespace + "/" + pod.Name + "/" + container.Name
 		contScore[i] = float64(scoreForEachNUMANode(container.Resources.Requests, allocatablePerNUMA, scorerFn, resourceToWeightMap))
+		klog.V(6).InfoS("container scope scoring", "container", identifier, "score", contScore[i])
+	}
+	finalScore := int64(stat.Mean(contScore, nil))
+	klog.V(4).InfoS("container scope scoring final node score", "finalScore", finalScore)
+	return finalScore, nil
+}
+
+func logNRT(desc string, nrtObj *topologyv1alpha1.NodeResourceTopology) error {
```

Comment:
I think it would be better to move this helper function to pluginhelper.go in order to keep things consistent with the implementation in https://github.com/kubernetes-sigs/scheduler-plugins/pull/354.

Reply:
Done. Thank you Swati for the review!
I presume this change will create a conflict once #354 get merged. I'll keep an eye on it.